feature/manager.go

```diff
@@ -63,14 +63,27 @@ func newManager(cfg Config, desc setDesc) (*Manager, error) {
 	}
 
 	// Now, remove any features as directed by the config.
-	for _, fv := range fsets {
+	for set, raw := range fsets {
 		if cfg.NoTLVOnion {
-			fv.Unset(lnwire.TLVOnionPayloadOptional)
-			fv.Unset(lnwire.TLVOnionPayloadRequired)
+			raw.Unset(lnwire.TLVOnionPayloadOptional)
+			raw.Unset(lnwire.TLVOnionPayloadRequired)
+			raw.Unset(lnwire.PaymentAddrOptional)
+			raw.Unset(lnwire.PaymentAddrRequired)
+			raw.Unset(lnwire.MPPOptional)
+			raw.Unset(lnwire.MPPRequired)
```

Comment:
This change looks like it belongs more to the first commit where the bits are added?

Reply:
we haven't begun to set `PaymentAddr` or `MPP` in any of our feature sets, that doesn't happen until the end. So this is the preemptive commit, that allows the one-line changes later